### PR TITLE
Fix casts warnings.

### PR DESCRIPTION
**Pull request by HBelusca**
There were 3 casts warnings, fixed now.

### CODE REVIEW

File: src/cascadia/TerminalApp/App.cpp [hunk 1]
```diff
@@ -710,7 +710,7 @@ namespace winrt::TerminalApp::implementation
         // we clamp the values to the range [0, tabCount) while still supporting moving
         // leftward from 0 to tabCount - 1.
         _SetFocusedTabIndex(
-            (tabCount + focusedTabIndex + (bMoveRight ? 1 : -1)) % tabCount
+            static_int<int>((tabCount + focusedTabIndex + (bMoveRight ? 1 : -1)) % tabCount)
```
Comment (zadjii-msft) [inline src/cascadia/TerminalApp/App.cpp]: This one should be `static_cast`

Comment (HBelusca) [reply src/cascadia/TerminalApp/App.cpp]: oh yes, typo 😃

**Comment by adiviness** [comment]
Please don't force push in the future, it makes the code reviews more difficult to track. We can do a squash when the merge happens :)

**Pull request merged**